### ISSUE TITLE
Scrolling performance slowed

### ISSUE THREAD

**Issue by obaskanderi** [suzuki-0000/SKPhotoBrowser#145]
It appears that #144 has greatly effected scrolling performance. The cause for the slow down is the result of adding padding to the displayed image.

Padding Code in SKZoomingScrollView.displayImage(complete flag: Bool):
```
// create padding
let width: CGFloat = image.size.width + SKPhotoBrowserOptions.imagePaddingX
let height: CGFloat = image.size.height + SKPhotoBrowserOptions.imagePaddingY;
UIGraphicsBeginImageContextWithOptions(CGSizeMake(width, height), false, 0.0);
let context: CGContextRef = UIGraphicsGetCurrentContext()!;
UIGraphicsPushContext(context);
let origin: CGPoint = CGPointMake((width - image.size.width) / 2, (height - image.size.height) / 2);
image.drawAtPoint(origin)
UIGraphicsPopContext();
let imageWithPadding = UIGraphicsGetImageFromCurrentImageContext();
UIGraphicsEndImageContext();

**Comment by suzuki-0000** [comment]
thanks issue. this is critical. I'll fixed it ASAP

**Comment by suzuki-0000** [comment]
I fixed this at latest mater and v3.1.2.
